src/settings-ui/Microsoft.PowerToys.Settings.UI.Library/Enumerations/ColorRepresentationType.cs

```diff
@@ -65,5 +65,17 @@ public enum ColorRepresentationType
         /// Color presentation as CIEXYZ color space (X[0..95], Y[0..100], Z[0..109]
         /// </summary>
         CIEXYZ = 10,
+
+        /// <summary>
+        /// Color presentation as RGB float (red[0..1], green[0..1], blue[0..1])
+        /// </summary>
+        RgbFloat = 11,
+
+        /// <summary>
+        /// Color presentation as integer decimal value 0-16777215
+        /// </summary>
+#pragma warning disable CA1720 // Identifier contains type name
+        Decimal = 12,
+#pragma warning restore CA1720 // Identifier contains type name
```

Comment:
We should use another name here.
Using names that can collide with type names doesn't sound like a good idea.

Reply:
with how the code works, i don't think it is that easy and it is just an enum.  Worse, that is what it actually is called.

Reply:
The name can be set to "Decimal" in `var decimalFormatName = ColorRepresentationType.Decimal.ToString();`, the change added to "src/settings-ui/Microsoft.PowerToys.Settings.UI.Library/ViewModels/ColorPickerViewModel.cs".
Just do `var decimalFormatName = "Decimal";` instead, or will this break the code elsewhere?

Reply:
i can try, the code was finicky in a few spots.

Reply:
got it, lotta touch points